metadataStore/test/test_commands.py

```diff
@@ -131,6 +131,12 @@ def test_begin_run():
     beamline_id = 'sample_beamline'
     yield _begin_run_with_cfg_tester, bcfg, time, beamline_id
 
+
+@context_decorator
+def _event_tester(descriptor, seq_num, data, time):
+    begin
```

Comment:
I was adding event tests to travis when Dan came over to do some changes to the schema. I will fix this while modifying commands.py and updating travis tests